src/storage/src/hummock/event_handler/uploader.rs

```diff
@@ -312,12 +453,62 @@ impl SealedData {
         }
     }
 
+    fn poll_success_merge_imm(&mut self, cx: &mut Context<'_>) -> Poll<Option<MergeImmTaskOutput>> {
```

Comment:
I think there is no any IO operation in the merge task. So the merge task should not get any error and should always return with success?

Reply:
Yes. Currently, the merging task would only return `Ok`, but I think it is not harmful to have a match arm for `Err` in this function.